src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -49,7 +49,7 @@ public sealed class RandomizedPcaTrainer : TrainerEstimatorBase<AnomalyPredictio
         internal const string Summary = "This algorithm trains an approximate PCA using Randomized SVD algorithm. "
             + "This PCA can be made into Kernel PCA by using Random Fourier Features transform.";
 
-        public class Arguments : UnsupervisedLearnerInputBaseWithWeight
+        public class Options : UnsupervisedLearnerInputBaseWithWeight
```

Comment:
>Options [](start = 21, length = 7)

xml docs are coming later?

Reply:
yeap. 

---
In reply to: [254353429](https://github.com/dotnet/machinelearning/pull/2390#discussion_r254353429) [](ancestors = 254353429)